module-system/sov-modules-wallet-blueprint/Cargo.toml

```diff
@@ -0,0 +1,43 @@
+[package]
+name = "sov-modules-wallet-blueprint"
```

Comment:
Why do we need a new crate? Can we just put the `snap-wallet` in `sov-modules-rollup-blueprint` so everything is in one place?

Reply:
That would bring more feature complexity to `sov-modules-rollup-blueprint` as I need to differentiate when we generate for the CLI (so we use things like `clap` as dependency), and for the WASM (that is as minimal as possible to keep the binary size small)